src/main/component/Editor.svelte

```diff
@@ -90,10 +96,21 @@
       editorRef.setContent(value);
       text = editorRef.getContent({format: 'text'});
     }
+    if (editorRef && readonly !== readonlyCache) {
+      readonlyCache = readonly;
+      if (typeof editorRef.mode?.set === 'function' && isDisabledOptionSupported(editorRef)) {
+        editorRef.mode.set(readonly ? 'readonly' : 'design');
+      } else {
+        interface TinyMCEEditor4 extends TinyMCEEditor {
+          setMode: (input: string) => void
+        }
+        (editorRef as TinyMCEEditor4).setMode(disabled ? 'readonly' : 'design');
```

Comment:
This throws an error when using TinyMCE < 7.6 as it only exists on TinyMCE 4

Reply:
removed